File: output/CodeLlama/codenet/Python/Java/s759547301.java
import java.util.Scanner;

public class s759547301 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        String S = sc.nextLine();

        if (S.equals("Sunny")) {

            System.out.println("Cloudy");

        }

        if (S.equals("Cloudy")) {

            System.out.println("Rainy");

        }

        if (S.equals("Rainy")) {

            System.out.println("Sunny");

        }

    }

}

File: output/ChatGPT/codenet/Python/C++/s759547301.cpp
#include <iostream>
using namespace std;

int main() {
    string S;
    cin >> S;
    
    if (S == "Sunny") cout << "Cloudy" << endl;
    
    if (S == "Cloudy") cout << "Rainy" << endl;
    
    if (S == "Rainy") cout << "Sunny" << endl;
    
    return 0;
}